integrationTests/multiShard/relayedTx/common.go

```diff
@@ -73,6 +74,29 @@ func CreateAndSendRelayedAndUserTx(
 	return relayedTx
 }
 
+// CreateAndSendRelayedAndUserTxV2 will create and send a relayed user transaction for relayed v2
+func CreateAndSendRelayedAndUserTxV2(
+	nodes []*integrationTests.TestProcessorNode,
+	relayer *integrationTests.TestWalletAccount,
+	player *integrationTests.TestWalletAccount,
+	rcvAddr []byte,
+	value *big.Int,
+	gasLimit uint64,
+	txData []byte,
+) *transaction.Transaction {
+	txDispatcherNode := getNodeWithinSameShardAsPlayer(nodes, relayer.Address)
+
+	userTx := createUserTx(player, rcvAddr, value, 0, txData)
+	relayedTx := createRelayedTxV2(txDispatcherNode.EconomicsData, relayer, userTx, gasLimit)
+
+	_, err := txDispatcherNode.SendTransaction(relayedTx)
+	if err != nil {
```

Comment:
Isn't an assert for nil useful here?

Reply:
this is the same behavior as for the relayed tx integration test. I assume it is enough to only print the error if any. the test would fail anyway if this doesn't work